ui-ngx/src/app/shared/pipe/custom-translate.pipe.ts

```diff
@@ -0,0 +1,48 @@
+///
+/// Copyright © 2016-2023 The Thingsboard Authors
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+
+import { Pipe, PipeTransform } from '@angular/core';
+import { i18nPrefix, i18nRegExp } from '@shared/models/constants';
+import { TranslateService } from '@ngx-translate/core';
+import { isString } from '@core/utils';
+
+@Pipe({
+  name: 'customTranslate'
+})
+export class CustomTranslatePipe implements PipeTransform {
```

Comment:
This pipe not support translate params

Reply:
Is it possible to set a default translation for this pipe if the text is not translated?